src/effects/effectprocessor.h

```diff
@@ -32,7 +35,7 @@ class EffectProcessor {
     // samples. The group provided allows the effect to maintain state on a
     // per-group basis. This is important because one Effect instance may be
     // used to process the audio of multiple channels.
-    virtual void process(const QString& group,
+    virtual void process(const ChannelHandle& group,
```

Comment:
And updated ChannelHandleAndGroup to be uniformly referred to as handle_group to hint at its pair-like nature.